modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -1353,6 +1353,33 @@ public void testUpdateCustomObjectField() throws Exception {
 				relatedObjectDefinition.getObjectDefinitionId(),
 				"r_relationship_" + objectDefinition.getPKObjectFieldName());
 
+		long relationshipObjectFieldId =
+			relationshipObjectField.getObjectFieldId();
+
+		// Object field relationship dbColumName cannot end with underline
+
+		relationshipObjectField = _updateCustomObjectField(
+			relationshipObjectField,
```

Comment:
This field belongs to a definition already published, right?
I don't think this test passes through the new code you added in the previous work (the test does not cover the new code).

Reply:
Hi @marcelabc,
I ran debug and saw that it is passing the new code, the validation I put in was [this one](https://github.com/liferay/liferay-portal/blob/85e754bf50b9d7391f12806bc84116d52cd27cb6/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java#L641-L645).

Reply:
You're correct. I misunderstood :+1: